File: src/main/java/net/thedragonteam/armorplus/util/EnchantmentUtils.java
/*
 * Copyright (c) TheDragonTeam 2016-2017.
 */

package net.thedragonteam.armorplus.util;

import net.minecraft.enchantment.Enchantment;

import static net.minecraft.enchantment.Enchantment.*;

/**
 * @author Sokratis Fotkatzikis - TheDragonTeam
 **/
public class EnchantmentUtils {

    public static Enchantment getEnchantment(int nameOrId) {
        return getEnchantmentByID(nameOrId);
    }

    public static Enchantment getEnchantment(String nameOrId) {
        return getEnchantmentByLocation(nameOrId);
    }

    public static int getID(Enchantment enchant) {
        return getEnchantmentID(enchant);
    }
}

File: src/main/java/net/thedragonteam/armorplus/util/EnchantmentUtils.kt
/*
 * Copyright (c) TheDragonTeam 2016-2017.
 */

package net.thedragonteam.armorplus.util

import net.minecraft.enchantment.Enchantment

import net.minecraft.enchantment.Enchantment.*

object EnchantmentUtils {

    fun getEnchantment(name: String): Enchantment = getEnchantmentByLocation(name) as Enchantment

    fun getEnchantment(id: Int): Enchantment = getEnchantmentByID(id) as Enchantment

    fun getID(enchant: Enchantment): Int = getEnchantmentID(enchant)
}
